fuse_examples/tests/test_notebook_hello_world.py

```diff
@@ -0,0 +1,16 @@
+import os
+import unittest
+from testbook import testbook
+
+class NotebookHelloWorldTestCase(unittest.TestCase):
+    
+    def test_example_2(self):
```

Comment:
The test takes a long time.
It might be the CPU - maybe the default should be the GPU0 (see multimodal notebook) and keep in comment the steps to move it to CPU?

Reply:
Indeed long time. I'll change it